dapr-http-client/src/Dapr/Client/HttpClient/Types/Binding.hs

```diff
@@ -0,0 +1,18 @@
+module Dapr.Client.HttpClient.Types.Binding where
+
+import Dapr.Client.HttpClient.Types.Internal
+import Data.Aeson
+import Data.Map as Map
+import GHC.Generics (Generic)
+
+newtype Metadata = Metadata (Map String String) deriving (Eq, Show, Generic, ToJSON)
```

Comment:
There's already a RequestMetadata.

Reply:
Changed it to Core module's RequestMetadata type.